src/lib/edge-functions/proxy.js

```diff
@@ -91,8 +93,11 @@ const initializeProxy = async ({
 
     if (!registry) return
 
-    // Setting header with geolocation.
+    // Setting header with geolocation and site info.
     req.headers[headers.Geo] = JSON.stringify(geoLocation)
+    req.headers[headers.Site] = Buffer.from(
```

Comment:
I would maybe extract this into a separate function that returns the value of the header and has these ops in separate stages (i.e. creating the object, JSON-stringifying, then base64-encoding).

But that's totally not a blocker, and we can always revisit that later. Let's ship! 🚀

Reply:
Yeah! And then we'd be able to unit test it ✨